### ISSUE TITLE
It only works for the current request

### ISSUE THREAD

**Issue by waltermaldonado** [felixkiss/uniquewith-validator#103]
I am not sure if this is a bug or I missed something. I have a unique validation in a field named 'code' together with a field named 'team_id'. The thing is: when I save the record, I only send in the request the code field, and it is failing validation, saying there is a field with this value already. How can I overcome this?

Thanks in advance.

**Comment by robinhoo1973** [comment]
Yes, looks like the code parameters in the validator only accept first 2 params. Not sure it's bug of laravel?

**Comment by waltermaldonado** [comment]
> Yes, looks like the code parameters in the validator only accept first 2 params. Not sure it's bug of laravel?

I don't think so, since laravel does not have a unique_with validator (see https://laravel.com/docs/5.8/validation#available-validation-rules)

**Comment by robinhoo1973** [comment]
Yes, you’re right, I just found it’s another issue caused

waltermaldonado <notifications@github.com>于2019年3月13日 周三上午7:36写道：

**Comment by hellohasan** [comment]
@waltermaldonado hey, how you overcome that, I am facing this issue now.
